package.json

```
@@ -6,6 +6,11 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```

According to the README, babel is handling the build, but while you're doing dev, it's nice to have the type checker running (`npm run type-check:watch`). You'd also want the type checker to run as part of the build (`npm run type-check`).

```suggestion
     "test": "echo \"Error: no test specified\" && exit 1",
     "type-check": "tsc",
     "type-check:watch": "npm run type-check -- --watch",
  },
```

---

I would hold off on this until there's an actual `build` or `start` script because right now (for the scope of this PR), it's too early to add such scripts.